src/polyfills.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+/** @fileoverview @suppress {suspiciousCode} */
```

Comment:
suspiciousCode indeed.

Reply:
@jridgewell wrote the `if(false && ...)` hack. Perhaps it can be removed?